#
# K2HDKC DBaaS based on Trove
#
# Copyright 2020 Yahoo Japan Corporation
#
# K2HDKC DBaaS is a Database as a Service compatible with Trove which
# is DBaaS for OpenStack.
# Using K2HR3 as backend and incorporating it into Trove to provide
# DBaaS functionality. K2HDKC, K2HR3, CHMPX and K2HASH are components
# provided as AntPickax.
#
# For the full copyright and license information, please view
# the license file that was distributed with this source code.
#
# AUTHOR:   Hirotaka Wakabayashi
# CREATE:   Mon Sep 14 2020
# REVISION:
#

diff -rc /opt/stack/trove/integration/scripts/files/elements/ubuntu-guest/install.d/98-ssh /opt/stack/trove/integration/scripts/files/elements/centos-8-guest/install.d/98-ssh
*** /opt/stack/trove/integration/scripts/files/elements/ubuntu-guest/install.d/98-ssh   2020-09-13 07:06:08.851039227 +0900
--- /opt/stack/trove/integration/scripts/files/elements/centos-8-guest/install.d/98-ssh 2020-09-13 12:16:16.378262786 +0900
***************
*** 5,8 ****
  set -e
  set -o xtrace

! dpkg-reconfigure openssh-server
--- 5,8 ----
  set -e
  set -o xtrace

! #dpkg-reconfigure openssh-server

